README.md

```diff
@@ -211,34 +221,75 @@ Link to other services:
 docker-compose -f docker-compose.yaml -f docker-compose.link.yaml up
 ```
 
+# How to get an output
+
+There are 4 possible outcomes depending on the input:
+
+1. The input is a valid payment file.
+Outcome:
+    - The transactions are logged,
+    - The payment file is moved from `inbound` to `archive`,
+    - Each valid payment request is sent to the Topic beginning with `ffc-pay-request`.
```

Comment:
Do we want to name the Topics as it couples the service?

We could use their configured env var name instead, i.e. for this one its `PAYMENT_TOPIC_ADDRESS`

We could change the actual value of this and it would still be correct

Look in the `docker-compose.yaml` file to find these :D